openedx/adg/common/mailchimp_pipeline/apps.py

```diff
@@ -0,0 +1,11 @@
+"""
```

Comment:
Suggestion: Can we update location of mailchimp_pipeline app to openedx/adg/common**/lib/**mailchimp_pipeline. It seems sort of lib and we have separate directory for that.